<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayTeams</class>
 <widget class="QDialog" name="DisplayTeams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>651</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Summary">
    <attribute name="title">
     <string>Summary</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser_Summary">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>591</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Summary_Loc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File location:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Detailed">
    <attribute name="title">
     <string>Detailed</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser_Detailed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>591</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Detailed_Loc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File location:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PPID Reports</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>The reports have been saved to text files for future convenience.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
